Rocket.Chat/Models/Subscription/UnmanagedSubscription.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  UnmanagedSubscription.swift
+//  Rocket.Chat
+//
+//  Created by Samar Sunkaria on 8/10/18.
+//  Copyright © 2018 Rocket.Chat. All rights reserved.
+//
+
+import Foundation
+import DifferenceKit
+
+struct UnmanagedSubscription: UnmanagedObject, Equatable {
```

Comment:
If we're trying to abstract Realm away from most of the code, shouldn't we rename the Realm type to `ManagedSubscription` and call this one `Subscription`?

Reply:
Right now, it does not feel like we are completely abstracting away realm, especially since we do maintain a reference to the realm object and use it quite often. Once we get Sourcery to write all out unmanaged objects for us, only then should we perform this change.